config/locales/fr.yml

```diff
@@ -2046,8 +2046,8 @@ webauthn_platform_recommended.description_save_time: Save time by using your fac
 webauthn_platform_recommended.heading: Set up face or touch unlock for a quick and easy sign in
 webauthn_platform_recommended.skip: Skip
 webauthn_setup_mismatch.description_undo: Cliquez sur « Annuler » pour supprimer cette option.
-webauthn_setup_mismatch.description.webauthn: Nous avons remarqué que vous utilisiez une clé de sécurité au lieu du déverrouillage facial ou tactile. Cliquez sur « Suite » pour utiliser votre clé de sécurité afin de vous connecter à partir de maintenant.
-webauthn_setup_mismatch.description.webauthn_platform: Nous avons remarqué que vous utilisiez le déverrouillage facial ou tactile au lieu d’une clé de sécurité. Cliquez sur « Suite » pour utiliser le déverrouillage facial ou tactile afin de vous connecter à partir de maintenant.
+webauthn_setup_mismatch.description.webauthn_html: Nous avons remarqué que vous utilisiez une clé de sécurité au lieu du déverrouillage facial ou tactile. <br/>Cliquez sur « Suite » pour utiliser votre clé de sécurité afin de vous connecter à partir de maintenant.
```

Comment:
Had to do it this way because it looks like based on spacing, we had the right amount of spacing, but it looks like ruby cleans it up. I would welcome any other suggestion for how to break the line.

Reply:
Yes we've used `&nbsp;` in other cases, which I think would be preferable because `<br />` can cause strange line breaks depending on how text collapses at different screen sizes.

Reply:
Hmm i tried that, It just ended up. making a weird indentation since it doesnt move the first << to the line. looks like this. 
![Screenshot 2025-02-24 at 1 15 07 PM](https://github.com/user-attachments/assets/04052859-da49-4a6d-a624-09fba9175e3e)

Reply:
Can you push those changes or share a diff so we can see if there might be some other issue causing the indentation?

Reply:
That would be a design/content change we'd want to run past the UX folks.

I'm still not sure I follow the issues with the `&nbsp;` that you were having.

Reply:
I can push up the change with nbsp snad see if that makes sense.